content/source/docs/cloud/api/admin/organizations.html.md

```diff
@@ -151,7 +160,101 @@ Resource Name | Description
 curl \
   --header "Authorization: Bearer $TOKEN" \
   --header "Content-Type: application/vnd.api+json" \
-  https://app.terraform.io/api/v2/admin/organizations/my-organization
+  https://tfe.example.com/api/v2/admin/organizations/my-organization
+```
+
+### Sample Response
+
+```json
+{
+  "data": {
+    "id": "my-organization",
+    "type": "organizations",
+    "attributes": {
+      "access-beta-tools": false,
+      "external-id": "org-XBiRp755dav5p3P2",
+      "is-disabled": false,
+      "name": "my-organization",
+      "terraform-build-worker-apply-timeout": null,
+      "terraform-build-worker-plan-timeout": null,
+      "terraform-worker-sudo-enabled": false,
+      "notification-email": "my-organization@example.com",
+      "global-module-sharing": false,
+      "sso-enabled": false
+    },
+    "relationships": {
+      "owners": {
+        "data": [
+          {
+            "id": "user-hxTQDETqnJsi5VYP",
+            "type": "users"
+          }
+        ]
+      },
+      "subscription": {
+        "data": null
+      },
+      "feature-set": {
+        "data": null
+      }
+    },
+    "links": {
+      "self": "/api/v2/admin/organizations/my-organization"
+    }
+  }
+}
+```
+
+## Update an Organization
+
+`PATCH /organizations/:organization_name`
+
+Parameter            | Description
+---------------------|------------
+`:organization_name` | The name of the organization to update
+
+Status  | Response                                        | Reason
+--------|-------------------------------------------------|----------
+[200][] | [JSON API document][] (`type: "organizations"`) | The organization was successfully updated
+[404][] | [JSON API error object][]                       | Organization not found or user unauthorized to perform action
+[422][] | [JSON API error object][]                       | Malformed request body (missing attributes, wrong types, etc.)
+
+
+### Request Body
+
+This PATCH endpoint requires a JSON object with the following properties as a request payload. Note that the Admin Organizations API may offer a different set of attributes than the [Organizations API](/docs/cloud/api/organizations.html#request-body-1).
+
+Key path                                               | Type    | Default   | Description
+-------------------------------------------------------|---------|-----------|------------
+`data.type`                                            | string  |           | Must be `"organizations"`
+`data.attributes.access-beta-tools`                    | boolean | false     | Whether or not workspaces in the organization can be configured to use beta versions of Terraform.
+`data.attributes.global-module-sharing`                | boolean | false     | If true, modules in the organization's private module repository will be available to all other organizations in this TFE instance.
```

Comment:
```suggestion
`data.attributes.global-module-sharing`                | boolean | false     | If true, modules in the organization's private module repository will be available to all other organizations in this TFE instance. Enabling this will disable any previously configured [module consumers](./module-sharing.html).
```